NEWS.txt

```diff
@@ -3,12 +3,82 @@ Release Notes
 =============
 
 
+Theano 0.9.0rc1 (15th of February, 2017)
+========================================
+
+This release extends the 0.9.0beta1 and announces the upcoming final release 0.9.
+
+Highlights (since 0.9.0beta1):
+ - Better integration of Theano+libgpuarray packages into conda distribution
+ - Better handling of Windows end-lines into C codes
+ - Better compatibility with NumPy 1.12
+ - Faster scan optimizations
+ - Fixed broadcast checking in scan
+ - Bug fixes related to merge optimizer
+ - many other bug fixes and improvements
+ - Updated documentation
+
+ - Old GPU back-end:
+   * In MRG, replaced method `multinomial_wo_replacement()` with new method `choice()`
+
+ - New GPU back-end:
+   * Value of a shared variable is now set inplace
+
+A total of 24 people contributed to this release, see the list at the bottom.
+
+Convolution updates:
+ - Implement conv2d_transpose convenience function
+
+GPU:
+ - GPUMultinomialFromUniform op now supports multiple dtypes
+
+New features:
+ - OpFromGraph now allows gradient overriding for every input
+ - Added Abstract Ops for batch normalization that use cuDNN when available and pure Theano CPU/GPU alternatives otherwise
+ - Added new Theano flag cuda.enabled
+ - Added new Theano flag print_global_stats to print some global statistics (time spent) at the end
+
+Others:
+ - Split op now has C code for CPU and GPU
+ - "theano-cache list" now includes compilation times
+
+Other more detailed changes:
+ - Changed optdb.max_use_ratio to 6
```

Comment:
Remove this. This is too much a detail and the only consequence is less warning output during tests.